plotting-visualization/dicomtree.py

```diff
@@ -0,0 +1,90 @@
+"""
+sudo apt install tix-dev
+sudo pip install -U pydicom
+python3 dicomtree.py file.dcm
+=========================================
+Show a dicom file using hierarchical tree
+=========================================
+Show a dicom file using a hierarchical tree in a graphical window.
+"""
+
+# authors : Guillaume Lemaitre <g.lemaitre58@gmail.com>
+# license : MIT
+
```

Comment:
Please just remove the author here - we don't record them in files any more.  In any case, Guillaume was just the latest, in setting it up for online examples.  Suggest a single comment line at top with ``` # Copyright pydicom authors 2019.  See LICENSE file for details```, as is done in all pydicom files.